src/applications/vaos/appointment-list/components/cards/confirmed/AppointmentInstructions.jsx

```diff
@@ -9,7 +12,14 @@ export default function AppointmentInstructions({ instructions }) {
 
   return (
     <div className="vads-u-flex--1 vads-u-margin-bottom--2 vaos-u-word-break--break-word">
-      <h4 className="vaos-appts__block-label">{header}</h4>
+      {isHomepageRefresh && (
+        <h2 className="vaos-appts__block-label vads-u-font-size--h4">
+          {header}
+        </h2>
+      )}
+      {!isHomepageRefresh && (
+        <h4 className="vaos-appts__block-label">{header}</h4>
```

Comment:
What's the reason that this element is an h4, but the other version (line 16) is an h2? Seems like they should be the same heading level

Reply:
These components are used on different pages, which different headings, so the level needs to fit within the heading order of the page it's being used on.